### DISCUSSION TITLE
OSD status shows down while OSD pod is up and running. How to identifying root-cause by interpreting "ceph -s" command output

### DISCUSSION THREAD

**Discussion by chintha-v** [rook/rook#10004] [Q&A]
Hi @travisn,
We installed rook-ceph with 2 OSDs. Suddenly one OSD went to down state while it's corresponding OSD pods is up and running. We could see some degrading statements in "ceph -s" but are not able to interpret. Could you please help us in identifying root-cause for osd down by interpreting following output. We have been collecting "ceph osd df" and "ceph -s" command output every minute.

**"ceph osd df"  command output:**

**Tue Apr 5 23:33:18**
ID CLASS WEIGHT REWEIGHT SIZE RAW USE DATA OMAP META AVAIL %USE VAR PGS STATUS
0 ssd 0.19530 1.00000 200 GiB 111 GiB 110 GiB 438 KiB 1.3 GiB 89 GiB 55.59 1.00 33 up
1 hdd 0.19530 1.00000 200 GiB 111 GiB 110 GiB 16 KiB 1.1 GiB 89 GiB 55.52 1.00 32 up
				TOTAL 400 GiB 222 GiB 220 GiB 455 KiB 2.4 GiB 178 GiB 55.55
MIN/MAX VAR: 1.00/1.00 STDDEV: 0.04

**Tue Apr 5 23:34:21** 
ID CLASS WEIGHT REWEIGHT SIZE RAW USE DATA OMAP META AVAIL %USE VAR PGS STATUS
0 ssd 0.19530 1.00000 200 GiB 111 GiB 110 GiB 438 KiB 1.3 GiB 89 GiB 55.60 1.00 33 up
1 hdd 0.19530 1.00000 200 GiB 111 GiB 110 GiB 16 KiB 1.2 GiB 89 GiB 55.54 1.00 0 down
				TOTAL 400 GiB 222 GiB 220 GiB 455 KiB 2.4 GiB 178 GiB 55.57
MIN/MAX VAR: 1.00/1.00 STDDEV: 0.03

**"ceph -s" command output:**

**Tue Apr 5 23:33:18**
cluster:
id: 24704d74-9bbc-40b2-b4dc-cb487aad234e
health: HEALTH_WARN
clock skew detected on mon.b
mons a,c are low on available space
Degraded data redundancy: 28825/86477 objects degraded (33.333%), 32 pgs degraded, 32 pgs undersized
175 slow ops, oldest one blocked for 810 sec, mon.b has slow ops

services:
mon: 3 daemons, quorum a,b,c (age 6d)
mgr: a(active, since 6d)
osd: 2 osds: 2 up (since 6d), 2 in (since 6d)

data:
pools: 2 pools, 33 pgs
objects: 28.83k objects, 110 GiB
usage: 222 GiB used, 178 GiB / 400 GiB avail
pgs: 28825/86477 objects degraded (33.333%)
32 active+undersized+degraded
1 active+clean

**Tue Apr 5 23:34:21**
cluster:
id: 24704d74-9bbc-40b2-b4dc-cb487aad234e
health: HEALTH_WARN
clock skew detected on mon.b
mons a,c are low on available space
1 osds down
1 host (1 osds) down
Degraded data redundancy: 57650/86477 objects degraded (66.665%), 32 pgs degraded
195 slow ops, oldest one blocked for 875 sec, mon.b has slow ops

services:
mon: 3 daemons, quorum a,b,c (age 6d)
mgr: a(active, since 6d)
osd: 2 osds: 1 up (since 41s), 2 in (since 6d)

data:
pools: 2 pools, 33 pgs
objects: 28.83k objects, 110 GiB
usage: 222 GiB used, 178 GiB / 400 GiB avail
pgs: 96.970% pgs not active
57650/86477 objects degraded (66.665%)
32 undersized+degraded+peered
1 active+clean

Thank you so much in advance!

**Comment by travisn** [comment]
The issue may be clock skew in the mons
```
clock skew detected on mon.b
```
You'll want to configure ntp to make sure the clocks are sync'd. After this is resolved I would expect the OSDs to behave better.
